app/lib/max_allowed_days_loader.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module MaxAllowedDaysLoader
+  module_function
+
+  def load_account_suspension
+    config[:account_suspension]
+  end
+
+  def load_account_inactivity
+    config[:account_inactivity]
+  end
+
+  def load_contract_unpaid_time
+    config[:contract_unpaid_time]
+  end
+
+  def config
+    @config ||= load_config
+  end
+
+  def valid_configuration?
+    config && @valid_configuration
+  end
+
+  def load_config
+    config = (ThreeScale.config.max_allowed_days || {}).each_with_object({}) do |(key, value), collection|
+      collection[key.to_sym] = value.public_send(:days) if value.to_i >= 1
```

Comment:
We do not need that. at all

Reply:
@hallelujah We don't need what exactly?

Reply:
Ok, I got my answer in https://github.com/3scale/porta/pull/574#discussion_r254209038

Reply:
agree.